packages/odyssey-react/.storybook/main.js

```diff
@@ -11,10 +11,44 @@ module.exports = {
       propFilter: (prop) => (prop.parent ? !/node_modules/.test(prop.parent.fileName) : true),
     },
   },
-  "addons": [
+  addons: [
     "@storybook/addon-links",
     "@storybook/addon-essentials",
-    "@storybook/preset-scss",
-    "@storybook/addon-a11y"
-  ]
-}
+    "@storybook/addon-a11y",
+  ],
+  webpackFinal(config) {
+    config.module.rules.push(
+      {
+        test: /\.scss$/,
+        use: [
+          'style-loader',
+          {
+            loader: 'css-loader',
+            options: {
+              importLoaders: 2,
+              sourceMap: true,
+              modules: {
+                auto: true,
+                localIdentName: "[folder]__[local]__[hash:base64:5]",
```

Comment:
Is there specific rationale for changing our selector naming convention from `.ods-component--child` to `.Component__someChild`?

Reply:
@edburyenegren-okta no rationale! This code is just for the storybook examples and isn't (necessarily) going to be reflective of the class names we ship from compiled components via [OKTA-416031](https://oktainc.atlassian.net/browse/OKTA-416031). These classes here can be anything because the expectation is that cascade based overrides are no longer going to be a part of how customized UI is created.

I would think that JIRA above would be a great place for us to finalize the structure of class names we want to ship for the simplest adoption patterns of importing components as ES modules directly.

Reply:
If there is no practical reason for the change, I ask that we revert it to the existing `--` convention until we discuss it in the ticket and finalize it as a group. I would rather be consistent in these changes and discuss and implement changes intentionally.

Reply:
@arnoldsandoval-okta sure. I can make this look a bit closer to what we had before although keep in mind it will not match 100% because class names are generated programmatically now rather than by hand. I'd also reiterate that class names for storybook aren't critical because there is not guarantee or expectation that someone will be using the cascade to make customizations going forward.